### ISSUE TITLE
[Feature Request] make GNOME 40 dash separator available for < 3.38 through dash-to-dock

### ISSUE THREAD

**Issue by hrdwrrsk2** [micheleg/dash-to-dock#1393]
this issue may duplicate of #622, but separator feature in GNOME 40 actually to separate unpinned running applications between pinned ones (similar to Windows 10X/ChromeOS taskbar compared to macOS dock i think)

will be dash-to-dock v70 bring GNOME 40 dash separator to old GS ? it will be nice, but it would be great if also used for separating pinned apps/show apps icon with trash and mounted volume icons (and unpinned running apps, ofc)

**Comment by nestormarius** [comment]
+1
We need GNOME 40 support please!

**Comment by hrdwrrsk2** [comment]
> +1
> We need GNOME 40 support please!

yes i also waiting for it, but for discussion about GNOME 40 support just go #1389 or #1358 instead. this is about new dash separator feature should be backported to old GNOME shell

**Comment by fvrlak** [comment]
Had the same issue. Found a workaround and turned it into a small CLI tool:                                                                                                                                                                 
                                                                                                                                                                                                                                              
  https://github.com/fvrlak/gnome-taskbar-separator
                                                                                                                                                                                                                                              
  Creates vertical line separators between pinned app groups. Works on GNOME 46+, Zorin OS, Ubuntu.
